server/datastore/mysql/apple_mdm.go

```diff
@@ -4167,7 +4167,12 @@ WHERE
 		return nil, ctxerr.Wrap(ctx, err, "get mdm config assets by name")
 	}
 
-	return res, nil
+	assetMap := map[fleet.MDMAssetName]fleet.MDMConfigAsset{}
```

Comment:
the only reason I didn't do this is to add another signal when the db doesn't have all the values, just in case the caller does a `len` check.

having said that, and considering other notes you left about this I will do an explicit check here!

Reply:
ohhh, thank you! TIL